DashKitUI/Views/ScheduledExpirationReminderEditView.swift

```diff
@@ -39,14 +45,25 @@ struct ScheduledExpirationReminderEditView: View {
     var content: some View {
         VStack {
             RoundedCardScrollView(title: LocalizedString("Scheduled Reminder", comment: "Title for scheduled expiration reminder edit page")) {
-                RoundedCard {
-                    RoundedCardValueRow(
-                        label: LocalizedString("Scheduled Reminder", comment: "Label for scheduled expiration reminder row"),
-                        value: dateFormatter.string(from: selectedDate),
-                        highlightValue: true
-                    )
-                    DatePicker("", selection: $selectedDate, in: allowedReminderDateRange)
-                        .datePickerStyle(WheelDatePickerStyle())
+                if self.horizontalSizeClass == .compact {
+                    // Keep picker outside of card in compact view, because it
```

Comment:
[nit] "because it" what?

Reply:
... forces full device width.  Will update, thanks.